src/modules/endpoint/MemoryDatabase.js

```diff
@@ -163,6 +165,30 @@ class MemoryDatabase {
   languagesLoaded = () => this._languages !== null
   eventsLoaded = () => this._events !== null
 
+  writeLastUpdate = async () => {
```

Comment:
I think we should use the `cities.json` to store the data: https://github.com/Integreat/integreat-react-native-app/blob/e8004719f66bd2e807a7eb27ad88240ad1ef31bb/docs/11-persistence.md#L17

I think a field last_updated would fit there. We should probably only store this in `cities.json` and load it to a map "cites_code -> last_updated".

It makes sense to have the last_updated timestamp of all the offline cities in memory. This will allow us to display the timestamps in the settings e.g.

Reply:
I think it makes sense to do this in this task as it is not that big.

Reply:
This should be discussed.

Reply:
Makes also sense for me to put it into the cities.json

Reply:
Done.